torch_geometric/nn/aggr/basic.py

```diff
@@ -36,6 +37,18 @@ def forward(self, x: Tensor, index: Optional[Tensor] = None, *,
         return self.reduce(x, index, ptr, dim_size, dim, reduce='min')
 
 
+class MulAggregation(Aggregation):
+    def forward(self, x: Tensor, index: Optional[Tensor] = None, *,
+                ptr: Optional[Tensor] = None, dim_size: Optional[int] = None,
+                dim: int = -2) -> Tensor:
+        if ptr is not None:  # TODO
+            import warnings
+            warnings.warn(f"'{self.__class__.__name__}' with "
```

Comment:
I'd prefer raising a `NotImplementedError` here

Reply:
Thanks, @Padarn. Sounds fair. Will change it.